x/cron/types/params.go

```diff
@@ -0,0 +1,96 @@
+package types
+
+import (
+	"fmt"
+	// errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	// sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+var _ paramtypes.ParamSet = (*Params)(nil)
+
+var (
+	DefaultSecurityAddress = []string{"aib1sdggcl7eanaanvcsmvars7l0unsge65wzjm3dc"} //assuming BECH32_PREFIX=aib, will need a Default admin to whitelist contract for cron operations
```

Comment:
this one shouldn't be kept as aib1sdggcl7eanaanvcsmvars7l0unsge65wzjm3dc, the original rollapp-wasm is used for muliple rollapp and the contract prefixes is different. You can leave it on init genesis or on your own init scripts

Reply:
sure, thanks